### DISCUSSION TITLE
Uncertainty of collected data

### DISCUSSION THREAD

**Discussion by masoudsouri** [iiasa/CWatM#114] [General]
What is the uncertainty associated with the collected data, for example, in the case of lakes and reservoirs?

**Comment by mikhailsmilovic** [comment]
Guten Tag Masoudsouri,

Each global input dataset will be different. Lakes and reservoirs come from https://www.hydrosheds.org/products/hydrolakes. This includes the 

- georeferenced spatial extent, 
- maximum area, 
- maximum volume, 
- average discharge, and 
- year of construction (if applicable). 

These static inputs for lake and reservoir input values are probably more missing than incorrect. Perhaps uncertainty can be quantified by how long the product has been publicly available and, therefore, under new critical eyes.

**Reply by masoudsouri** [reply]
Dear Dr. Smilovic,
Guten tag
I appreciate your prompt response and the valuable references you provided. The discussion we had proved to be insightful and beneficial.
